package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "5.9.10",
+  "version": "5.10.0",
```

Comment:
I'm adding something significant, so while the code is backward-compatible, it's a milestone (and resets the patch version so it doesn't grow too large).

Reply:
Is there anything bad with having a high patch version?

Reply:
Not really.

But we had one UI go to patch version 39 or something and that just begs the question of whether we did anything significant in there or was it all bugfixes 😁